sycl/unittests/SYCL2020/GetNativeOpenCL.cpp

```diff
@@ -133,13 +130,12 @@ TEST(GetNative, GetNativeHandle) {
 
   get_native<backend::opencl>(Context);
   get_native<backend::opencl>(Queue);
-  get_native<backend::opencl>(Program);
   get_native<backend::opencl>(Device);
   get_native<backend::opencl>(Event);
   get_native<backend::opencl>(Buffer);
 
   // Depending on global caches state, piDeviceRetain is called either once or
   // twice, so there'll be 6 or 7 calls.
```

Comment:
Do we need to update those magic numbers now?

Reply:
Thanks, done in 5056a12b9b47343502036f670e2e2b24fff34f37